src/components/search/advancedSearch/fieldInput/TextInputs.js

```diff
@@ -20,10 +20,9 @@ import { LogicalOperatorsEnum } from "@/components/search/enums";
  * @param {Object} props
  * @returns {React.JSX.Element}
  */
-function FieldInput({ key, index, fieldValue, doAdvancedSearch }) {
```

Comment:
så skal du nok også rette denher:
<FieldInput
            key={`inputField-${index}`}
            index={index}
            fieldValue={field}
            doAdvancedSearch={doAdvancedSearch}
          />
altså fjerne key .. hvis nu <FileldInput> selv laver den :)

Reply:
Jeg startede med at fjerne den, men lint brokker sig. Folk skriver at React vil have key på et komponent for at optimere performance. (selvom vi giver den et key inde i selve komponenten 🙃)